2.js

```diff
@@ -0,0 +1,117 @@
+/*
+    #1
+
+var a = 1, b = 1, c, d;
+c = ++a; alert(c);         =>   a=a+1; c=a   (a=2;c=2)
+d = b++; alert(d);         =>   d=b; b=b+1   (d=1;b=2)
+c = (2+ ++a); alert(c);    =>   a=a+1; c=2+a (a=3;c=5)
+d = (2+ b++); alert(d);    =>   d=2+b; b=b+1 (d=4;b=3)
+alert(a);                  =>   a=3
+alert(b);                  =>   b=3
+
+    #2
+
+var a = 2;
+var x = 1 + (a *= 2);    => a=a*2 ; x=1+a  (a=4;x=5)
+
+*/
+var res,oper,a,b,c,d;
+
+a = +prompt('Введите первое число');
+b = +prompt('Введите второе число');
+if (a >= 0 && b >= 0) {
+    res = a - b;
+} else if (a < 0 && b < 0) {
+    res = a * b;
+} else if ((a >= 0 && b < 0) || (a < 0 && b >= 0)) {
```

Comment:
Здесь можно оставить просто else и результат будет то же
```suggestion
} else {
```